ui/main/src/app/modules/share/pipes/opfab-title-case.pipe.ts

```diff
@@ -0,0 +1,27 @@
+/* Copyright (c) 2022, Alliander (http://www.alliander.com)
+/* Copyright (c) 2018-2022, RTE (http://www.rte-france.com)
```

Comment:
For the moment, RTE has not updated the file so the line "...2018-2022, RTE..." should be removed.